i18n/de.json

```diff
@@ -1594,8 +1594,11 @@
   "channel_modal.purposeEx": "Z.B.: \"Ein Kanal um Fehler und Verbesserungsvorschläge abzulegen\"",
   "channel_modal.type": "Typ",
   "channel_notifications.allActivity": "Für alle Aktivitäten",
-  "channel_notifications.globalDefault": "Globaler Standard ({notifyLevel})",
-  "channel_notifications.ignoreChannelMentions": "Ignore mentions for @channel, @here and @all",
+  "channel_notifications.globalDefault": "Globaler Standard",
```

Comment:
These translations need to be reviewed as I've entered them using a translation tool for testing purposes.